Can't use release aab with yaml

```

##[debug]loading INPUT_METADATAROOTPATH
##[debug]loading INPUT_SHOULDUPLOADMAPPINGFILE
##[debug]loading INPUT_MAPPINGFILEPATH
##[debug]loading INPUT_VERSIONCODEFILTERTYPE
##[debug]loading ENDPOINT_AUTH_7a37d71f-d930-44fb-afea-5ca9e3f7bc82
##[debug]loading ENDPOINT_AUTH_SCHEME_7a37d71f-d930-44fb-afea-5ca9e3f7bc82
##[debug]loading ENDPOINT_AUTH_PARAMETER_7a37d71f-d930-44fb-afea-5ca9e3f7bc82_USERNAME
##[debug]loading ENDPOINT_AUTH_PARAMETER_7a37d71f-d930-44fb-afea-5ca9e3f7bc82_PASSWORD
##[debug]loading ENDPOINT_AUTH_SYSTEMVSSCONNECTION
##[debug]loading ENDPOINT_AUTH_SCHEME_SYSTEMVSSCONNECTION
##[debug]loading ENDPOINT_AUTH_PARAMETER_SYSTEMVSSCONNECTION_ACCESSTOKEN
##[debug]loading SECRET_SYSTEM_ACCESSTOKEN
##[debug]loaded 23
##[debug]Agent.ProxyUrl=undefined
##[debug]Agent.CAInfo=undefined
##[debug]Agent.ClientCert=undefined
##[debug]Agent.SkipCertValidation=undefined
##[debug]task result: Failed
##[error]Unhandled: Unexpected identifier
##[debug]Processed: ##vso[task.issue type=error;]Unhandled: Unexpected identifier
##[debug]Processed: ##vso[task.complete result=Failed;]Unhandled: Unexpected identifier
Finishing: Publish to production on Play Store

```

Am seeing the same issue - Google Play - Release Bundle.

Seeing nothing useful in the logs - even when turned up to verbose output, I'm new to this plugin - has it ever worked?

`2019-11-01T20:34:31.7937252Z ==============================================================================
2019-11-01T20:34:31.7937307Z Task         : Google Play - Release Bundle
2019-11-01T20:34:31.7937335Z Description  : Release an app bundle to the Google Play Store
2019-11-01T20:34:31.7937378Z Version      : 3.160.0
2019-11-01T20:34:31.7937404Z Author       : Microsoft Corporation
2019-11-01T20:34:31.7937432Z Help         : ms-resource:loc.helpMarkDown
2019-11-01T20:34:31.7937459Z ==============================================================================
2019-11-01T20:34:32.0218064Z ##[debug]agent.TempDirectory=d:\a\_temp
2019-11-01T20:34:32.0248368Z ##[debug]loading inputs and endpoints
2019-11-01T20:34:32.0254740Z ##[debug]loading ENDPOINT_AUTH_PARAMETER_SYSTEMVSSCONNECTION_ACCESSTOKEN
2019-11-01T20:34:32.0267573Z ##[debug]loading ENDPOINT_AUTH_SCHEME_SYSTEMVSSCONNECTION
2019-11-01T20:34:32.0271170Z ##[debug]loading ENDPOINT_AUTH_SYSTEMVSSCONNECTION
2019-11-01T20:34:32.0273545Z ##[debug]loading INPUT_APPLICATIONID
2019-11-01T20:34:32.0275674Z ##[debug]loading INPUT_AUTHTYPE
2019-11-01T20:34:32.0277333Z ##[debug]loading INPUT_BUNDLEFILE
2019-11-01T20:34:32.0279068Z ##[debug]loading INPUT_CHANGELOGFILE
2019-11-01T20:34:32.0280784Z ##[debug]loading INPUT_LANGUAGECODE
2019-11-01T20:34:32.0282512Z ##[debug]loading INPUT_MAPPINGFILEPATH
2019-11-01T20:34:32.0284435Z ##[debug]loading INPUT_METADATAROOTPATH
2019-11-01T20:34:32.0286128Z ##[debug]loading INPUT_ROLLOUTTOUSERFRACTION
2019-11-01T20:34:32.0287928Z ##[debug]loading INPUT_SERVICEACCOUNTKEY
2019-11-01T20:34:32.0289521Z ##[debug]loading INPUT_SHOULDATTACHMETADATA
2019-11-01T20:34:32.0291166Z ##[debug]loading INPUT_SHOULDUPLOADMAPPINGFILE
2019-11-01T20:34:32.0292653Z ##[debug]loading INPUT_TRACK
2019-11-01T20:34:32.0294816Z ##[debug]loading INPUT_USERFRACTION
2019-11-01T20:34:32.0296295Z ##[debug]loading INPUT_VERSIONCODEFILTERTYPE
2019-11-01T20:34:32.0316480Z ##[debug]loaded 17
2019-11-01T20:34:32.0334342Z ##[debug]Agent.ProxyUrl=undefined
2019-11-01T20:34:32.0336162Z ##[debug]Agent.CAInfo=undefined
2019-11-01T20:34:32.0336784Z ##[debug]Agent.ClientCert=undefined
2019-11-01T20:34:32.0339335Z ##[debug]Agent.SkipCertValidation=undefined
2019-11-01T20:34:32.0569707Z ##[debug]task result: Failed
2019-11-01T20:34:32.0626742Z ##[error]Unhandled: Unexpected identifier`

--- a/Tasks/google-play-release-bundle/GooglePlay.ps1
+++ b/Tasks/google-play-release-bundle/GooglePlay.ps1
@@ -1,7 +1,7 @@
 param (
     [string]$serviceAccountKey,
     [string]$serviceEndpoint,
-    [string]$packageName
+    [string]$packageName,
     [string]$bundleFile,
     [string]$track,
     [string]$userFraction,

